mt9t112 camera needs 100 milliseconds for initializing Special thanks to Phil

Signed-off-by: Kuninori Morimoto <morimoto.kuninori@renesas.com>
Reported-by: Phil Edworthy <Phil.Edworthy@renesas.com>
---
 drivers/media/video/mt9t112.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/media/video/mt9t112.c b/drivers/media/video/mt9t112.c index 7438f8d..e581d8a 100644
--- a/drivers/media/video/mt9t112.c
+++ b/drivers/media/video/mt9t112.c
@@ -885,7 +885,7 @@ static int mt9t112_s_stream(struct v4l2_subdev *sd, int enable)
 		/* Invert PCLK (Data sampled on falling edge of pixclk) */
 		mt9t112_reg_write(ret, client, 0x3C20, param);
 
-		mdelay(5);
+		mdelay(100);
 
 		priv->flags |= INIT_DONE;
 	}
--
1.6.3.3


